Arriven/db1000n
# Reset metrics on config change

# Description

When the config is updated with new targets, metrics remain the same hence meaningless.
My little proposal is to reset metrics on each config update to get relevant information about current targets.
Sorry if anything is wrong, I learned Go yesterday.

Fixes # (issue)
N/A
## Type of change


## How Has This Been Tested?

- Run the program with config A
- Update config
- Observe `Traffic` and `ProcessedTraffic` metrics are reset.

![IMG_0070](https://user-images.githubusercontent.com/13631794/160242157-f68cea8a-3b71-4499-a571-9f887db60a46.jpeg)
